### DISCUSSION TITLE
v1.3.0 queries

### DISCUSSION THREAD

**Discussion by Darshan-Ramesh** [Lightning-AI/pytorch-lightning#7482] [General]
Hello Team,

I recently moved from v1.1.1 to v1.3.0 and I observe a few things.

a. I use wandb as the logger. 
b. I use model checkpoint callback to monitor my validation loss with verbose set to True.
c. I log the training and validation loss and f1 scores only on epochs like the following image

`self.log('train_loss',loss,prog_bar=True,on_step=False, on_epoch=True)
`
\

1. In v1.1 I was able to see the verbose while in this version, during training or validation, I do not see any information/verbose.  
<img src="https://user-images.githubusercontent.com/63796397/117811520-f27fd300-b260-11eb-9d49-f64c22ee88aa.PNG "/>

\
2. I now found a new metric, global_step being logged in wandb as seen in the attached images, which is not really clear.
<img src="https://user-images.githubusercontent.com/63796397/117811515-f1e73c80-b260-11eb-8874-8b62043900f3.PNG " width="500" height="400"/>
\
3. Also, why epoch is being logged in this manner?
<img src="https://user-images.githubusercontent.com/63796397/117811073-6ff71380-b260-11eb-951f-e6a2f16e14a0.PNG " width="500" height="400"/>
\
Could someone help me understand what might be the reasons for all the above observations?

**Comment by Darshan-Ramesh** [comment]
Got it the "checkpoint_callback" is bool now, which wasn't in v1.1.1 so the verbose is working for me. I am still waiting for the answers as to why epochs are plotted in a step-by-step way and not a straight line.
